js/fc.js

```diff
@@ -39,6 +39,7 @@ var FAILSAFE_CONFIG;
 var RXFAIL_CONFIG;
 var PID_ADVANCED_CONFIG;
 var FILTER_CONFIG;
+var SPECIAL_PARAMETERS;
```

Comment:
but the configurator would still be broken for folks on 2.9.0 though, right? i think it's better to just make it work :)